src/test/java/io/reactivex/observers/TestObserverTest.java

```diff
@@ -958,13 +958,15 @@ public void assertValueSequence() {
             throw new RuntimeException("Should have thrown");
         } catch (AssertionError ex) {
             // expected
```

Comment:
let's remove this then?

Reply:
I say let's keep it to indicate catching te AssertionError is not an accident.

Reply:
Usually that's done by naming exception as `expected`:

```java
try {
  // …
  fail();
} catch (AssertionError expected) {
  // Assert exception message, etc.
}
```

Reply:
Fine.